glib/src/variant.rs

```diff
@@ -1218,7 +1218,11 @@ impl ToVariant for std::ffi::OsString {
 #[cfg(unix)]
 impl FromVariant for std::ffi::OsString {
```

Comment:
And also further above the `Path` conversions need to consider long paths on Windows like in `path_to_c()`

Reply:
Calling `path_to_c` literally should just work. I guess. I'll use `os_str_to_c` to convert `OsStr` to avoid that when not requested.